src/NuGet.Core/NuGet.Packaging/Rules/UnrecognizedLicenseIdentifierRule.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace NuGet.Packaging.Rules
 {
-    class UnrecognizedLicenseIdentifierRule : IPackageRule
+    internal class UnrecognizedLicenseIdentifierRule : IPackageRule
```

Comment:
see this [comment](https://github.com/NuGet/NuGet.Client/pull/3226#discussion_r377183570)

Reply:
[Rule #5](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md)